internal/api/plugin.go

```diff
@@ -105,3 +199,253 @@ func (s *Server) DeletePlugin(c echo.Context) error {
 
 	return c.NoContent(http.StatusNoContent)
 }
+
+func (s *Server) GetCategories(c echo.Context) error {
+	categories, err := s.db.FindCategories(c.Request().Context())
+	if err != nil {
+		message := echo.Map{
```

Comment:
can you also update these to follow the new Pattern ? 
use `NewErrorResponse("message")`

Reply:
done